osutil/epoll/epoll_test.go

```diff
@@ -204,12 +205,26 @@ func (*epollSuite) TestEpollWaitEintrHandling(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(e.RegisteredFdCount(), Equals, 1)
 
-	mockReturnedN := 0
-	mockReturnedErr := unix.EINTR
+	var mu sync.Mutex
+	eintr := true
+	doReturnEintr := func() bool {
+		mu.Lock()
+		defer mu.Unlock()
+		return eintr
+	}
+	stopEintr := func() {
+		mu.Lock()
+		defer mu.Unlock()
+		eintr = false
+	}
```

Comment:
(extreme nitpick) I don't love `doReturnEintr` as a name, it was confusing to me at first how the function fit in. Maybe something more like a question verb would be clearer to me, like `shouldReturnEintr`.

Feel free to totally disregard this though, it's just naming preferences.